### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by MrPoofPoof**
Finished code refractoring

### CODE REVIEW

File: TimerGUI.pyw [hunk 1]
```diff
@@ -1,120 +1,97 @@
...
+# !!!CHANGE NOT ADVISED!!! time before "change log file" info pop's out again
+timerDelay = 3600
 
-#first window creator 
-def createInitialWindow():
-    #first window constructor
-    initialWindow = tk.Tk()
-    initialWindow.title('Timer lasera')
+#GUI and commands code
+import tkinter as tk, time, math
```
Comment (KFrydryk) [inline TimerGUI.pyw]: Place imports on top of the file, before defining any constants/variables

File: TimerGUI.pyw [hunk 2]
```diff
@@ -1,120 +1,97 @@
...
-        initialWindow,
-        text = 'Czy włączono logger lasera?',
-        width = 22,
-        height = 2,
-        font='Times 18 bold',
-        padx=10,
-        pady=10,
-        justify='center'
-        )
+    isLoggerOn = createLabel(initialWindow, 'Czy włączono logger lasera?')
```
Comment (KFrydryk) [inline TimerGUI.pyw]: In python strings are generally used with double quotes for text, and single quotes for identifiers/paths. No functional difference, just good practice.

File: TimerGUI.pyw [hunk 3]
```diff
@@ -1,120 +1,97 @@
...
     isLoggerOn.pack()
 
-    timerStartButton = tk.Button(
-        initialWindow,
-        text='TAK +1h',
-        bg='light green',
-        justify='center',
```
Comment (KFrydryk) [inline TimerGUI.pyw:54]: i.e. this is a proper use of single quoted string

File: TimerGUI.pyw [hunk 4]
```diff
@@ -1,120 +1,97 @@
...
-    shortWindowHideButton = tk.Button(
-        initialWindow,
-        text='NIE +2 min',
-        bg='red',
-        font='Times 18 bold',
-        command = shortWindowHide
-    )
-    shortWindowHideButton.pack(side='right', expand= True,fill=tk.BOTH)
-    initialWindow.mainloop()
+    timerStartButton = createButton(initialWindow, f'TAK +{round((timerDelay/3600))}h', 'light green', timerStart)
```
Comment (KFrydryk) [inline TimerGUI.pyw]: Try `"TAK +{time}h".format(time=round((timerDelay/3600)))` for better readability

File: TimerGUI.pyw [hunk 5]
```diff
@@ -1,120 +1,97 @@
...
+    timerStartButton.pack(side = 'left', expand = True, fill = tk.BOTH)
 
+    shortWindowHideButton = createButton(initialWindow, f'NIE +{round((shortHideButtonDelay/60))}min', 'red', shortWindowHide)
```
Comment (KFrydryk) [inline TimerGUI.pyw]: ditto

File: TimerGUI.pyw [hunk 6]
```diff
@@ -1,120 +1,97 @@
...
-        recurringWindow,
-        text = 'ZMIEŃ LOG LASERA \n(nie nad linią!)',
-        width = 22,
-        height = 2,
-        font='Times 18 bold',
-        padx=10,
-        pady=10,
-        justify='center'
-        )
+    changeLog = createLabel(recurringWindow, 'ZMIEŃ LOG LASERA \n(nie nad linią!)')
```
Comment (KFrydryk) [inline TimerGUI.pyw]: ditto

File: TimerGUI.pyw [hunk 7]
```diff
@@ -1,120 +1,97 @@
...
+import tkinter as tk, time, math
 
-    #Set window to appear in the middle
+def createWindow():
+    
+    window = tk.Tk()
+    window.title('Timer lasera')
+    
     windowWidth = 330
     windowHeight = 122
```
Comment (KFrydryk) [inline TimerGUI.pyw:19]: Might be worth to pass width, height and position in a structure as parameter for a generic `createWindow` method

**Review by KFrydryk — CHANGES_REQUESTED** [review]

**Comment by KFrydryk** [comment]
Also, drop the "Minor change" and "revert" commits

File: TimerGUI.pyw (hunk 1 above)
Comment (MrPoofPoof) [reply TimerGUI.pyw]: Done

**Review by KFrydryk — APPROVED** [review]
LGTM

**Pull request merged**